Ohshie.MathGame/OhshieMathGame/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using OhshieMathGame;
+using OhshieMathGame.GameModes;
```

Comment:
Please remove unused usings: 
![image](https://user-images.githubusercontent.com/108711476/220532398-76bea412-1df7-463c-9058-503727a5dc34.png)

Reply:
Ok i think i pushed new commits on to the main csharpacademy github.. somehow. OK, i seriously need to watch and read up on how to use github lol, who knew that it would require so much brain effort.

Anyways:
i polished code as per csharpacademy standards as much as i could and also restructured it to use more methods instead of code.
then i expanded current record tracking system. (it did exist previously, but just displayed game data after player decides to stop playing current game mode). Now player can check his previous attempts until he closes the app.

as for division algorithm im not exactly sure what you mean, as i think the way division works rn is a good alternative to a simple int. Player must calculate his answer with .00 precision only and that allows for some nice equations.